### DISCUSSION TITLE
[Github] remote:Repository not found

### DISCUSSION THREAD

**Discussion by utnFBlasco** [dds-jv/foro#319] [Consultas de herramientas] [answered]
Buenas, cuando quise pullear el repo me emepezó a figurar el error de remote;Repository not found. Busque en Stack Overflow y dicen que es porque el repositorio es privado que realice git remote set-url https://usuario@url-repositorio. Lo hago y tira el mismo error. Adjunto algunas capturas a ver si alguien puede ayudarme con esto. Con otros repos funciona perfectamente.

![Untitled](https://user-images.githubusercontent.com/82328433/137161960-119eeadf-da5d-4c02-9db2-9a2cdf37f2d7.png)

**Comment by gastonprieto** [comment, marked as answer]
Buenas @utnFBlasco 

veo dos cosas distintas una es un clone y otra un pull, no me queda claro si el clone te funciono y luego no te funciono el pull.

Por lo pronto otro de stackoverflow dice que es tema de credenciales, fijate con [esto](https://stackoverflow.com/questions/37813568/git-remote-repository-not-found)

**Reply by utnFBlasco** [reply]
Gracias ya lo solucione con github desktop se me habian mezclado credenciales de la cuenta de la facu y la cuenta del laburo. Gracias
